parser/misc.go

```diff
@@ -557,6 +561,7 @@ var tokenMap = map[string]int{
 	"PLAN":                     plan,
 	"PLAN_CACHE":               planCache,
 	"PLUGINS":                  plugins,
+	"POINT":                    to_point,
```

Comment:
https://dev.mysql.com/doc/refman/8.0/en/gis-point-property-functions.html Since MySQL 8.0 already takes `point` to represent a point in X and Y coordinates, let's use another termination.

Reply:
OK. We have to avoid use `POINT` here